__fixtures__/test-project/web/src/components/Contact/Contacts/Contacts.tsx

```diff
@@ -27,19 +30,20 @@ const formatEnum = (values: string | string[] | null | undefined) => {
   }
 }
 
-const truncate = (text) => {
-  let output = text
-  if (text && text.length > MAX_STRING_LENGTH) {
+const truncate = (value: string | number) => {
+  let output = value.toString()
```

Comment:
Don't forget the same fix should be present in the text-project! :)

Reply:
Thanks, regen-ing test project